extensions/vault/spi/src/main/java/io/quarkus/agroal/VaultDbService.java

```diff
@@ -0,0 +1,12 @@
+package io.quarkus.agroal;
```

Comment:
the package should be ending with `.vault`, same for the `VaultDbService` since it is just the spi

Reply:
right. this was just an oversight of renaming agroal-spi as vault-spi. thanks for catching that.